Sources/StreamChatSwiftUI/ChatChannelList/ChatChannelListItem.swift

```diff
@@ -115,6 +109,22 @@ public struct ChatChannelListItem: View {
         }
     }
     
+    private var shouldShowReadEvents: Bool {
+        if let message = channel.latestMessages.first,
+           message.isSentByCurrentUser,
+           !message.isDeleted {
+            return channel.config.readEventsEnabled
+        }
+
+        return false
```

Comment:
I really like how this is written. Very clear and concise and understandable. Great job!